images/macos/tests/Common.Tests.ps1

```diff
@@ -85,4 +85,10 @@ Describe "VSMac" {
         $vsPath | Should -Exist
         $vstoolPath | Should -Exist
     }
-}
\ No newline at end of file
+}
+
+Describe "Swig" {

```

Comment:
we should either add Swig to macOS 10.14 or add a skip condition to the test.

Reply:
added for macOS 10.14